performances/performance_test/src/ros2/tracker.cpp

```diff
@@ -35,16 +29,23 @@ void performance_test::Tracker::scan(
 
     if (_tracking_options.is_enabled){
 	
+        std::map<std::string, TrackingNumber>::iterator it = _tracking_number_count_map.find(header.node_name);
```

Comment:
This could be using a `std::unordered_map`.
I think that otherwise the find and the string comparison may be a source of relevant overhead since it has to be done at every message received.